products/extensions/b2b-suite/guides/storefront/acl-routing.md

```diff
@@ -38,32 +27,32 @@ Shopware\B2B\AclRoute\Framework\AclRoutingUpdateService::create()
     ->addConfig($myAclConfig);
 ```
 
-This way you can easily create and store the resources. Of course in order to show a nice frontend you need to provide snippets for translation too. The snippets get automatically created from resource and privilege names and are prefixed with `_acl_`. So the resource `contingentgroup` needs a translation named `_acl_contingentgroup`.
+This way, you can easily create and store the resources. Of course, to show a nice frontend, you must also provide snippets for translation. The snippets get automatically created from resource and privilege names and are prefixed with `_acl_`. So the resource `contingentgroup` needs a translation named `_acl_contingentgroup`.
 
 ## Privilege names
 
 The default privileges are:
 
-| Privilege Name |                                    What it means                                    |
+| Privilege name |                                    What it means                                    |
 |----------------|:-----------------------------------------------------------------------------------:|
-| list           |                  entity listing, (e.g. indexActions, gridActions)                   |
-| detail         | disabled forms, lists of assignments, but only the inspection, not the modification |
-| create         |                              creation of new entities                               |
-| delete         |                            removal of existing entities                             |
-| update         |                         updating/Changing existing entities                         |
-| assign         |                        changing the assignment of the entity                        |
-| free           |                                   no restrictions                                   |
-
-It is quite natural to map CRUD Actions like this. However, assignment is a little less intuitive. This should help:
-
-* All assignment controllers belong to the resource of the right side of the assignment (e.g. `B2BContactRole` controller is part of the `role` resource).
-* All assignment listings have the detail privilege (e.g. `B2BContactRole:indexAction` is part of the `detail` privilege).
+| List           |                  Entity listing (e.g. indexActions, gridActions)                   |
```

Comment:
Also here, I suppose the casing of the privilege is important. It should probably even be in code fences.